interface/cassandra.thrift

```diff
@@ -25,7 +25,6 @@
 
 namespace java org.apache.cassandra.thrift
 namespace cpp org.apache.cassandra
-namespace csharp Apache.Cassandra
```

Comment:
The `csharp` generator was deprecated in thrift `0.13.0` and removed in `0.14.0`. We don't have any C# cassandra thrift clients, so it doesn't matter.